polaris-react/src/components/InContextLearning/InContextLearning.scss

```diff
@@ -1,11 +1,17 @@
+@import '../../styles/common';
+
 .InContextLearning {
   position: relative;
-  width: 250px;
+  width: 380px;
   border-radius: var(--p-border-radius-2);
   padding: var(--p-space-5);
   background-color: var(--p-surface);
   filter: drop-shadow(0px 0px 3px rgba(0, 0, 0, 0.1))
     drop-shadow(0px 4px 20px rgba(0, 0, 0, 0.15));
+
+  @include breakpoint-before(490px) {
```

Comment:
I'm following convention from another Polaris component that is referencing this sizing directly. Both Tag and TextField do this.